lib/matplotlib/rcsetup.py

```diff
@@ -944,6 +947,7 @@ def __call__(self, s):
     'examples.directory': ['', six.text_type],
 
     # Animation settings
+    'animation.html':         ['none', validate_movie_html_fmt],
```

Comment:
I guess this needs to be added to the rc template too. At least for now? A comment with an explanation about that this controls the _repr_html_ would also be good I think